applications/machine_learning/README.rst

```diff
@@ -223,6 +239,9 @@ The application supports the following build types:
 * ``ZDebug`` -- Debug version of the application - can be used to verify if the application works correctly.
 * ``ZRelease`` -- Release version of the application - can be used to achieve better performance and reduce memory consumption.
 
+Given board may also support some additional configurations of nRF Machine Learning application.
+For example, the nRF52840 Development Kit supports ``ZDebugNUS`` configuration that uses :ref:`nus_service_readme` instead of :ref:`zephyr:uart_api` for data forwarding.
```

Comment:
```suggestion
The given board can also support some additional configurations of the nRF Machine Learning application.
For example, the nRF52840 Development Kit supports ``ZDebugNUS`` configuration that uses :ref:`nus_service_readme` instead of :ref:`zephyr:uart_api` for data forwarding.
```
Where can I read more about this ZDebugNUS? It's appears out of the blue and is not listed above.

Reply:
Since application is quite flexible and introducing new configurations is easy, I would prefer not to list all of the supported configurations in docs. It would be quite difficult to maintain in the future I think.
The `ZDebugNUS` was mentioned as an example of a specific configuration and it's supported only on DK.

Reply:
Still, if you wish I will describe all of the board-specific configurations in details too

Reply:
Discussed offline. Follow up task is added.